docs/stellar-disbursement-platform/getting-started.mdx

```diff
@@ -235,26 +235,36 @@ In `docker-compose-sdp-anchor.yml`, update the following:
 <CodeExample>
 
 ```yaml
-# the public key of the account used for authentication
-# it should be the same public key for both attributes
-SEP10_ACCOUNT:
+# the public key of the Stellar account used for SEP-10 authentication
 SEP10_SIGNING_PUBLIC_KEY:
-# the private key of the account used for authentication
-# it should be the same secret key for both attributes
+#
+# the private key of the Stellar account used for SEP-10 authentication
+# it should be the same secret key for both attributes, in SDP and AP
 SEP10_SIGNING_PRIVATE_KEY:
 SECRET_SEP10_SIGNING_SEED:
-# the encryption key (any secret string) for the authentication token shared with wallets
-# it should be the same key for both attributes
-SEP24_JWT_SECRET:
+#
+# The secret used to sign the resulting SEP-10 JWT token. This should be a strong random string for encryption.
 SECRET_SEP10_JWT_SECRET:
-# the encryption key for the authentication token used for internal services
-# These could be different or the same value
-ANCHOR_PLATFORM_OUTGOING_JWT_SECRET:
+#
+# A shared encryption key used to sign JWT tokens in the SEP24 communications from AP->SDP.
+# The value needs to be the same for all three attributes below.
+SEP24_JWT_SECRET:
 SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET:
 SECRET_SEP24_MORE_INFO_URL_JWT_SECRET:
-# used to encrypt passwords of the SDP users
-# NOTE: you'll need to recreate any existing users in the database if this is reset
+#
+# A shared encryption key used to sign JWT tokens in the PlatformAPI communications from SDP->AP.
+# The value needs to be the same for both attributes below.
+ANCHOR_PLATFORM_OUTGOING_JWT_SECRET:
+SECRET_PLATFORM_API_AUTH_SECRET:
+#
+# The private key is used to sign JWT tokens for authenticating the requests incoming to the SDP. The Public key is used
+# to validate that the JWT token was signed by the SDP's private key. They can be generated with these commands:
+# openssl ecparam -name prime256v1 -genkey -noout -out ec_private_key.pem
```

Comment:
if you think it might be helpful to mention briefly what the EC256 encryption scheme is or its use case to provide context to those unfamiliar with it

Reply:
There's extensive content on elliptic curve encryption on the internet. The goal of this section is to help people configure their instances, rather than explaining these kinds of concepts that are broadly available on the internet.

Also, adding content into comments like we're doing in this section is just for straightforward directions.

Let me know if you think strongly otherwise, I can go back here and edit this section later.